src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/nrql-query-examples-mobile-monitoring.mdx

```diff
@@ -377,3 +377,39 @@ Below are some NRQL queries for common [handled exception](/docs/mobile-monitori
     ```
   </Collapser>
 </CollapserGroup>
+
+## MobileNetworkRequestSettings query examples
+
+Below are some common NRQL queries for the following mobile network requests:
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="hostnames"
+    title="Blocked or allowed hostnames"
+  >
+  Use case example...
+  ```sql
+    add query here
+  ```
```

Comment:
@asasha @CherylFrankenfield What's a use case for querying blocked/allowed hostnames?

Reply:
Pranav told me there are no sample queries for the network settings, so I'm closing out this thread and removing these additions.